phonecall_plan/__openerp__.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+# © 2016 6IT
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Phonecall plan',
+    'summary': "Wizard to plan calls from leads/partners",
+    'version': '8.0.1.0.0',
+    'depends': [
+        'base',
+        'crm'
+    ],
+    'author': 'Aurélien Rivet, 6IT, Odoo Community Association (OCA)',
```

Comment:
Here there should be only companies, nor individuals (unless you are on your own). I see 2 author entries here.